src/main/java/Duke.java

```diff
@@ -1,10 +1,228 @@
+        FileWriter fw = new FileWriter(filePath);
+        //System.out.println(tasks.length);
+        for (Task t: tasks) {
+            fw.write(t.toString()+"\n");
+        }
+        //fw.write(textToAdd);
+        fw.close();
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________\n"+
+                        "Hello! I'm Duke\n"+
+                        "What can I do for you?\n"+
+                        "____________________________________________________________");
+
+        try {
+            printFileContents("data/duke.txt");
+        } catch (FileNotFoundException e) {
+            System.out.println("task list doesn't exist yet");
+            createDirAndFile();
+        }
+
+        Scanner sc = new Scanner(System.in);
+
+        while(true){
+            String command = sc.nextLine();
+            if (command.equals("bye")) {
+                System.out.println("____________________________________________________________\n"+
+                                    "Bye. Hope to see you again soon!\n"+
+                                    "____________________________________________________________");
+                break;
+            }else if (command.equals("list")) {
+                System.out.println("____________________________________________________________");
+                listTasks();
+                System.out.println("____________________________________________________________");
+            }else if (command.contains("done")) {
+                doneTasks(command);
+                System.out.println("____________________________________________________________");
+            } else if (command.contains("todo")) {
+                System.out.println("____________________________________________________________");
+                todoTask(command);
+                System.out.println("____________________________________________________________");
+            }else if (command.contains("deadline")) {
+                System.out.println("____________________________________________________________");
+                deadlineTasks(command);
+                System.out.println("____________________________________________________________");
+            }else if (command.contains("event")) {
+                System.out.println("____________________________________________________________");
+                eventTask(command);
+                System.out.println("____________________________________________________________");
+            }else if (command.contains("delete")) {
```

Comment:
The spacing between the brackets and else if statement could be more consistent. See [layout](https://se-education.org/guides/conventions/java/basic.html#layout).